Lib/unittest/test/testmock/testpatch.py

```diff
@@ -1658,23 +1658,25 @@ def test_mock_calls_with_patch(self):
             finally:
                 p.stop()
 
+    @contextlib.contextmanager
+    def patch_sys_modules(self):
+        with swap_attr(sys, 'modules', sys.modules.copy()):
+            squizz = Mock()
+            sys.modules['squizz'] = squizz
+            yield squizz
```

Comment:
What exactly the method does is not very clear from the name. 
There are two places that need to patch in a mocked `squizz` module, but it seems to me that moving this into a method is too much indirection: the testcases would be much easier to read if they included the three lines directly even though it means a little code duplication:

        with swap_attr(sys, 'modules', sys.modules.copy()):
            squizz = Mock()
            sys.modules['squizz'] = squizz
            ...

Reply:
Resolved.